db/rdb.go

```diff
@@ -384,8 +384,9 @@ func (r *RDBDriver) CloseDB() (err error) {
 	return
 }
 
-// GetByCpeURI Select Cve information from DB.
-func (r *RDBDriver) GetByCpeURI(uri string) ([]models.CveDetail, error) {
+// getMatchingCpe returns matching CPEs  by psuedo-CPE
```

Comment:
__[Misspell]__ "psuedo" is a misspelling of "pseudo" ([link](https://sider.review/gh/repos/54628930/pulls/126#issue-3915066))

<sub>posted by Sider</sub>